embeds/ge2017-nugget/_source/main.js

```diff
@@ -1,88 +1,127 @@
-console.log('hellloo:');
-
-function findTargetContainer(container) {
-    var prev = container.previousElementSibling;
-    var classes = prev.classList;
-    if (classes.contains('fc-container--thrasher') || classes.contains('fc-container--video') || classes.contains('fc-container__mpu--mobile') || prev.getAttribute('id')=='manchester-attack' || prev.offsetHeight < 260) {
-        var target = findTargetContainer(prev);
-    } else {
-        var target = prev;
-    }
-    return target;
-}
+setTimeout(function() {
```

Comment:
We could avoid the need for a `setTimeout` by wrapping this code in an [IIFE](https://stackoverflow.com/questions/8228281/what-is-the-function-construct-in-javascript#8228308), which still avoids polluting the global namespace